api/src/main/java/org/pmiops/workbench/api/CloudTaskRdrExportController.java

```diff
@@ -46,11 +45,11 @@
    */
   @Override
   public ResponseEntity<Void> exportWorkspaceData(ArrayOfLong workspaceIds, Boolean backfill) {
-    if (workspaceIds == null || ((ArrayList) workspaceIds).isEmpty()) {
+    if (workspaceIds == null || workspaceIds.isEmpty()) {
       log.severe(" call to export Workspace Data had no Ids");
       return ResponseEntity.noContent().build();
     }
-    rdrExportService.exportWorkspaces(workspaceIds, Optional.ofNullable(backfill).orElse(false));
+    rdrExportService.exportWorkspaces(workspaceIds, BooleanUtils.isTrue(backfill));
```

Comment:
Hmm.. would it be more portable (and any less readable) to just do `Boolean.TRUE.equals(...)` ? This doesn't require a 3p library

Reply:
good point - updated